openhtf/core/test_state.py

```diff
@@ -380,13 +391,14 @@ def _finalize(self, test_outcome):
     aborting = test_outcome == test_record.Outcome.ABORTED
     assert not self.is_finalized or aborting, (
         'Test already completed with status %s!' % self._status.name)
-    # Sanity check to make sure we have a DUT ID by the end of the test.
-    if not self.test_record.dut_id:
-      raise BlankDutIdError(
-          'Blank or missing DUT ID, HTF requires a non-blank ID.')
 
     self.test_record.outcome = test_outcome
 
+    # Sanity check to make sure we have a DUT ID by the end of the test.
+    #if not self.test_record.dut_id:
+    #  raise BlankDutIdError(
+    #      'Blank or missing DUT ID, HTF requires a non-blank ID.')
+
```

Comment:
un-comment

Reply:
Ah, yes, trying to figure out why unit tests didn't work.